docs/troubleshooting/development/my-store-does-not-reflect-the-overrides-i-created.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "My store does not reflect the overrides I created"
+slug: "my-store-does-not-reflect-the-overrides-i-created"
+hidden: false
+createdAt: "2024-06-13T08:47:00.508Z"
+excerpt: "After creating a development workspace in a multi-language store, there is an error in my website."
+tags:
+    - faststore
+---
+
+During the [Section overriding process](https://developers.vtex.com/docs/guides/faststore/overrides-overview), you can experience issues such as overridden components not appearing or not behaving as you expected.
+
+Below are various troubleshooting checks and instructions you can use to solve your store's issue:
+
+| Issue | Troubleshooting instructions |
+| ----- | ---------------------------- |
+| Overrides fail to display in your store. | [Check the `getOverriddenSection` function](#checking-the-getoverriddensection-function). |
+| Overrides are not reflected in the Headless CMS. | [Remove default props](#removing-default-props). |
+| Styles are not being applied to the component. | [Check custom section class names](#checking-custom-section-class-names).|
+
+## Solution
+
+To understand and correct each error, see the solutions below:
+
+### Checking the `getOverriddenSection` function
+
+If you encounter an error message mentioning a missing function named `getOverriddenSection`, it means you need to import it into your code.
+
+This function is important for displaying product overrides within your store, allowing you to customize specific product page sections. Follow the steps below to import the function to your project:
+
+1. Open the file where you are overriding the section, for example, `ProductDetailsWithCustomButton.tsx`.
+2. At the top of the file, import the `getOverridenSetion` function:
+
+   ```tsx
+    import { getOverriddenSection } from '@faststore/core';
+    ```
+
+    Here's an example of how to use `getOverriddenSection` in your `ProductDetailsWithCustomButton.tsx` file:
+
+    ```tsx
+    import { ProductDetailsSection, getOverriddenSection } from '@faststore/core'
+
+    const ProductDetailsWithCustomButton = getOverriddenSection({
+    Section: ProductDetailsSection,
+    components: {
+        BuyButton: { Component: CustomBuyButton },
+    },
+    })
+    ```
+
+3. Run `yarn dev` to sync your changes with your development environment.
+
+### Adding the section’s schema in the Headless CMS
+
+If you can’t see the changes in the Headless CMS after [overriding a prop](https://developers.vtex.com/docs/guides/faststore/overrides-component-props) or a [section](https://developers.vtex.com/docs/guides/faststore/overrides-native-component), this may be due to a schema. The Headless CMS won’t display your custom section without one.
```

Comment:
```suggestion
If you cannot see the changes in the Headless CMS after [overriding a prop](https://developers.vtex.com/docs/guides/faststore/overrides-component-props) or a [section](https://developers.vtex.com/docs/guides/faststore/overrides-native-component), this may be due to a missing schema. The Headless CMS will not display your custom section without one.
```